.eslintrc.js

```diff
@@ -1,44 +1,30 @@
 module.exports = {
-    "env": {
-        "es6": true,
-        "node": true
+  env: {
```

Comment:
Why did you change this file? How it is relevant to your feature?

Reply:
I didn't change anything other than my IDE auto formats w.r.t. prettier style guide, so this file was simply autoindented.  It can be reverted back